storage_manager_new.py

```diff
@@ -5,184 +5,260 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.clear_self_items_list_and_load_items_from_file()
 
-    def load_items_from_file(self):
-        Item.instances = []
+    def clear_self_items_list_and_load_items_from_file(self):
+        self.items.clear()
         File = open(Storage.database_file, 'r')
         for line in File.readlines():
             line = line.rstrip('\n')
-            Item.instance_from_string(line)
-
-    def add_item(self):
-        # collect_new_item_data
-        new_items_data = []
-        add_more = True
-        while add_more == True:
-            new_item = Interface.get_user_choice(
-                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).')
-            if new_item.lower() != 'menu':
-                new_items_data.append(new_item)
-            else:
-                break
-            ask_for_more = Interface.get_user_choice(
-                'Would you like to add another item? Please type Y/N')
-            if ask_for_more.lower() == 'n':
-                add_more = False
-        # create_item_instance and write to database_file
-        for string in new_items_data:
-            try:
-                data = string.split(', ')
-                quantity = int(data.pop(3))
-                string_data = ', '.join(data)
-                for i in range(1, (quantity + 1)):
-                    Item.create_instance_from_string_and_write_to_database_file(
-                        string_data)
-            except IndexError:
-                pass
-
-    def show_all_items(self):
-        print('')
-        items_to_show = []
-        if Item.instances != []:
-            for item in Item.instances:
-                items_to_show.append(
-                    f'({item.category})  {item.type} {item.model}')
-            for item in sorted(items_to_show):
-                print(item)
-        else:
-            print('---\nList of items is empty\n---')
-        print('\n')
-
-    def show_items_from_category(self):
-        category = Interface.get_user_choice(
-            'Print items from category:').lower()
-        for item in Item.instances:
-            if category == item.category.lower():
-                print(f'({item.category})  {item.type} {item.model}')
-        print('\n')
-
-    def delete_item(self):
-        File = open(Storage.database_file, 'r')
-        items_in_file = File.readlines()
+            self.items.append(Item.create_instance_from_string(line))
+
+    def add_item(self, string_with_new_item_data):
+        try:
+            list_from_string_with_new_item_data = string_with_new_item_data.split(
```

Comment:
I wouldn't put types names in so many variables names (list and string) - I think `new_item_data = stringified_new_item_data` should be enough. Python is dynamically typed language, so by declaring types in variables names you kind of don't follow it/bypass around it. If you'd really like to know types of things, maybe you could use additional `typing` library that adds typing hints to your python code? I know this might be unclear now, just read about "statically/dynamically typed languages", also mentioned "typing" library and "static typing in python", "duck typing in python", this will clear things out a bit.